lib/omniauth/strategies/openid_connect.rb

```diff
@@ -111,10 +111,25 @@ def callback_phase
         fail!(:failed_to_connect, e)
       end
 
+      def other_phase
+        if logout_path_pattern.match(current_path)
```

Comment:
Please consider the following changes:

```ruby
def other_phase
  return call_app! unless logout_path_pattern.match(current_path)
  discover!
  redirect(end_session_uri) if end_session_uri  
end
```

Reply:
`call_app!` must be called when `end_session_uri` is falsy

I could change as the following (could be harder to read with multiple `return` and `call_app!` calls)
 
```
def other_phase
  return call_app! unless logout_path_pattern.match(current_path)
  discover!
  return redirect(end_session_uri) if end_session_uri
  call_app!
end
```